### ISSUE TITLE
TakePhotoAsync crash on LG V30.

### ISSUE THREAD

**Issue by JoseluDAM2000** [jamesmontemagno/MediaPlugin#708]
## Bug Information

Version Number of Plugin: **4.0.1.5**
Device Tested On: **LG V30**
Simulator Tested On:
Version of VS: **16.1.3 || 15.9.2**
Version of Xamarin: **3.5.0 || 4.0.0**
Versions of other things you are using: **Device Android Version 8.0.0**

I have been testing that on several phones and only fails on that one.

### Steps to reproduce the Behavior
Call the TakePhotoAsync.
Take a photo.

### Expected Behavior
The photo preview was shown.

### Actual Behavior
The camera activity crash. 
https://gist.github.com/JoseluDAM2000/063a26ebfd35a44921219a01b67da925

### Code snippet
```
string photoName = "IMG_" + DateTime.UtcNow.Ticks;
try
{
  file = await CrossMedia.Current.TakePhotoAsync(new Plugin.Media.Abstractions.StoreCameraMediaOptions
  {
    Name = photoName + ".jpg",
    Directory = "",
    SaveToAlbum = true,
    CompressionQuality = 75,
    CustomPhotoSize = 50,
    PhotoSize = PhotoSize.MaxWidthHeight,
    MaxWidthHeight = 2000,
    DefaultCamera = CameraDevice.Rear
  });
}
catch (Exception e2)
{
  return;
}
```

**Comment by jaytilly** [comment]
This looks like a dupe for https://github.com/jamesmontemagno/MediaPlugin/issues/586
 
PR for fix at https://github.com/jamesmontemagno/MediaPlugin/pull/715
Should be included with nuget: 4.3.1-beta

The fix is to first check that the intent extra needs to be set initially. If it doesn't need to be set, the putextra is skipped. When switching between back->front and vice-a-versa, the putextra works just fine...it was just the initial set when the extra is already false. Not sure what is going on under the LG covers but my guess is some state-machine is not happy.

This shouldn't have any negative side-effects and should fix the issues with the LG.

Tested on LG G7 that had the same problem.

**Comment by JoseluDAM2000** [comment]
By now the Xam.Plugin.Media 4.3.1 solves the problem 🎉 but i will wait until a full release to update it.
